cmd/analyze/main.go

```diff
@@ -244,4 +253,13 @@ func main() {
 		slog.InfoContext(ctx, "Starting dynamic analysis")
 		dynamicAnalysis(ctx, pkg, &resultStores)
 	}
+
+	return nil
+}
+
+func main() {
+	if err := run(); err != nil {
+		fmt.Fprintln(os.Stderr, err)
```

Comment:
nit: errors in go are usually uncaptalized fragements. You might want to print a prefix here: e.g. `"Error: "`.